### DISCUSSION TITLE
How to contribute to this project?

### DISCUSSION THREAD

**Discussion by GitHubUser-2024** [BitMaker-hub/NerdMiner_v2#534] [Q&A]
I may got the answer already, I cannot find it. Sorry if asking again.

Shall I make fork and open PR against original repo? Is this all what is needed?

**Comment by BitMaker-hub** [comment]
Hi all! Thanks for your question. I would like to organize  a little bit this part in order to let other contributors to work together in this project. 

The normal way is to make a PR... but could be interesting to be organized and think about new ideas
We used to have a Telegram chat with some contributors, this could still be used

And we could add some rules to other contributors who would like to be involved
